python/lib/eeg.py

```diff
@@ -722,7 +722,7 @@ def fetch_and_insert_event_files(
                     inheritance = True
 
                     if not event_metadata_file:
-                        message = '\nWARNING: no events metadata files (event.json) associated' +/
+                        message = '\nWARNING: no events metadata files (event.json) associated' +
                                   'with physiological file ID ' + physiological_file_id
```

Comment:
```suggestion
                        message = '\nWARNING: no events metadata files (event.json) associated' \
                                  'with physiological file ID ' + physiological_file_id
```

Try that instead maybe?